css/layout.css

```diff
@@ -31,4 +59,72 @@
   .column-half {
     width: 50%;
   }
+
+  img {
+    width: 100%;
+    height: 15.8rem;
+    object-fit: cover;
+    margin: 0.5rem 0 0.1rem 0;
+  }
+
+  .header-padding {
+    padding: 0.8rem 0;
+  }
+}
+
+img {
+  width: 100%;
+  height: 21.5rem;
+  object-fit: cover;
+  margin: 0.5rem 0 0.1rem 0;
+}
+
+.note-height {
+  height: 9.6rem;
+}
+
+.margin-zero {
+  margin: 0;
+}
+
+.header-font {
+  font-size: 1.3rem;
+}
+
+.button {
+  border: none;
+  border-radius: 0.2rem;
+  padding: 0.35rem 1.3rem;
+  margin: 1rem 0;
+  font-size: 1rem;
+}
+
+.font-bold {
+  font-weight: 700;
+}
+
+label {
+  font-size: 1.08rem;
+  margin: 1rem 0 0.7rem 0;
+}
+
+input,
+textarea {
+  height: 2.5rem;
+  margin: 0 0 0.8rem 0;
+  font-size: 1rem;
+  padding: 0.4 1rem;
+}
+
+.padding {
+  padding: 0.5rem;
+}
+
+.header-padding {
+  padding-top: 0.8rem;
+  padding-bottom: 0.8rem;
+}
+
+.row-reverse {
+  flex-direction: row-reverse;
 }
```

Comment:
You should _never_ have any general CSS rules below a media query. Make sure all general styling is laid out before media queries in the file, so that the media query can be allowed to take advantage of source order.